lib/active_admin/form_builder.rb

```diff
@@ -99,8 +99,8 @@ def has_many_actions(has_many_form, builder_options, contents)
       if builder_options[:sortable]
         has_many_form.input builder_options[:sortable], as: :hidden
 
-        contents << template.content_tag(:li, class: 'handle') do
-          "MOVE"
+        contents << template.content_tag(:li, class: 'handle sortable') do
```

Comment:
Adding the "sortable" seems unrelated to this PR, but I think it makes sense to me.

Reply:
Don't add this class so just remove the change please. It has been changed in #3862.

Reply:
Yes :) I added it just to make easier to style it

Reply:
Don't worry, we'll make sure it is in #3862 but we would prefer to avoid any changes here unless necessary and I don't think it is. I'll get that in #3862 for you. 😄

Reply:
ook thanks :)
removed in the next commit